content/_components/onedrive/index.md

```diff
@@ -6,152 +6,158 @@ description: This is the component for working with Microsoft OneDrive storage s
 
-## Triggers
+* **Enable File Attachments** - (checkbox, optional): If checked, file will be uploaded to local storage and link provided in response
+* **Time stamp field to poll on** - (dropdown, optional, default `Last Modified`): Select which date will be used to track files - `Last Modified` or `Created`
+* **Emit Behavior** - (dropdown, optional, default `Emit individually`): Defines the way result objects will be emitted, one of `Emit page` or `Emit individually`.
+* **Page Size** - (number, optional, defaults to 999, max 999): Indicates the size of pages to be fetched per request
+* **Start Time** - (string, optional): The timestamp to start polling from (inclusive) - using ISO 8601 Date time utc format - YYYY-MM-DDThh:mm:ssZ. Default value is the beginning of time (January 1, 1970 at 00:00).
+* **End Time** - (string, optional): The timestamp to stop polling (exclusive) - using ISO 8601 Date time utc format - YYYY-MM-DDThh:mm:ssZ. Default value is flow execution time.
 
-### Get New And Updated Files Polling
+#### Input Metadata
 
-Triggers to get all new and updated files since last polling. Polling is provided by `lastModifiedDateTime` file's property.
+There is no Input or Output metadata in this trigger.
 
-![Get New And Updated Files Polling](img/get-new-and-update.png)
+#### Output Metadata
 
-#### List of Expected Config fields
+Depends on `Enable File Attachments` and `Emit behavior` fields.
+ * If `Emit behavior` field is equal to `Emit page` - object with property `results` that contains array of files
+ * If `Emit behavior` field is equal to `Emit individually`, file information will fulfill whole message
+ * If `Enable File Attachments` checked, for each file there will be additional field - `attachmentUrl`
 
-* **Drive Identity** - OneDrive instance to work with.
-* **Folder path** - Dropdown with folders to poll files from.
-* **Emit Behavior** -  Available options: default is `Emit Individually` emits each object in separate message, `Fetch All` emits all objects in one message.
-* **Start Time** - Start datetime of polling. Default min date: `-271821-04-20T00:00:00.000Z`.
-* **End Time** - End datetime of polling. Default max date: `+275760-09-13T00:00:00.000Z`.
-* **Size Of Polling Page** - Indicates the size of pages to be fetched. Defaults to `1000`.
-* **Expand Children** - Checkbox, trigger retrieves files from subfolders of chosen path, if enabled. Disabled by default.
-* **Enable File Attachments** - Checkbox for attaching files content to response. Disabled by default.
 
-## Actions
-
-### Get File
+#### Limitations
 
-Action to get item from OneDrive by provided path in selected disc.
+* OneDrive API doesn't support filtering - as result we collect information about all files from selected folders and filter them locally (inside component) for each trigger execution
+* From point above, option `Emit page` not always emit records according to `Page Size`
```

Comment:
* From the point above, option `Emit page` does not always emit records according to `Page Size`